src/GrumPHP/Task/Blacklist.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Collection\FilesCollection;
+use GrumPHP\Exception\RuntimeException;
+
+/**
+ * Blacklist task
+ *
+ * @author  Igor Mukhin <igor.mukhin@gmail.com>
+ */
+class Blacklist extends AbstractExternalTask
+{
+    const COMMAND_NAME = 'git';
+
+    /**
+     * @return array
+     */
+    public function getDefaultConfiguration()
+    {
+        return array(
+            'keywords' => array(
+                'var_dump(',
+                'die('
+            ),
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getCommandLocation()
+    {
+        return $this->externalCommandLocator->locate(self::COMMAND_NAME);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function run(FilesCollection $files)
+    {
+        $files = $files->name('*.php');
+        if (0 === count($files)) {
+            return;
+        }
+
+        $config = $this->getConfiguration();
+        if (empty($config['keywords'])) {
+            throw new RuntimeException(
+                "Your grumphp config file misconfigued. You have empty keywords list under blacklist task."
+            );
+        }
+
+        $this->processBuilder->setArguments(array(
+            'git',
```

Comment:
This is because the `setArguments()` overwrites all arguments. The command is set as an attribute internally, so it will be overwritten. You could use the 'add()' method instead.